source/Playnite/Localization/LocSource.xaml

```diff
@@ -542,6 +542,9 @@ If disabled, games that use any item in any filter will be included in the view.
     <sys:String x:Key="LOCGameIsUnInstalledTitle">Not installed</sys:String>
     <sys:String x:Key="LOCGameHiddenTitle">Hidden</sys:String>
     <sys:String x:Key="LOCGameFavoriteTitle">Favorite</sys:String>
+    <sys:String x:Key="LOCGameHdrTitle">HDR</sys:String>
+    <sys:String x:Key="LOCGameHdrTooltip">If enabled, HDR will be enabled on the primary display before starting the game</sys:String>
+    <sys:String x:Key="LOCGameHdrNotSupportedTooltip">HDR is not supported on the primary display</sys:String>

```

Comment:
As my previous comment, I don't think this is needed.

Reply:
I've kept this. See https://github.com/JosefNemec/Playnite/pull/3290#discussion_r1105450181 for why